alphazero/utils/_utils.py

```diff
@@ -0,0 +1,9 @@
+import numpy as np
+
+
+def sample_tensor_indices(tensor, n_samples):
+    ind_1d = tensor.float().flatten().multinomial(n_samples)
+    samples = np.unravel_index(ind_1d, tensor.size())
+    samples = np.array(samples)
+    samples = [tuple(r) for r in samples.T]
```

Comment:
do we need the transpose here?

Reply:
To get the correct dimension it's needed. Can flip it some other way but will be more convoluted

Reply:
oh gotcha. I was just unsure what the dims of `samples` is, I thought if you iterated over it you'd just get each sample (and not the list of all samples like I'm guessing what you'd get without the transpose)